platform-sdk/swirlds-jasperdb/src/main/java/com/swirlds/merkledb/MerkleDbStatistics.java

```diff
@@ -240,6 +249,10 @@ public void registerMetrics(final Metrics metrics) {
                     metrics,
                     DS_PREFIX + COMPACTIONS_PREFIX + LEVEL_PREFIX + level + "_hashesSavedSpaceMb_" + label,
                     "Saved space during compactions of level %s, hashes store, %s, Mb".formatted(level, label)));
+            hashesStoreTotalSpaceMbList.add(buildDoubleAccumulator(
+                    metrics,
+                    DS_PREFIX + COMPACTIONS_PREFIX + LEVEL_PREFIX + level + "_hashesTotalSpaceMb_" + label,
```

Comment:
"total space size in Mb" is probably not the best name suffix for the new metrics. We already have metrics named "files size in Mb" (e.g. `hashesStoreFileSizeMb`). These two names look too similar. What about "file size by level in Mb" (e.g. `hashesFileSizeByLevelMb`)?

Reply:
Sure, I'll update the name

Reply:
Done